example/extension.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: operator.gardener.cloud/v1alpha1
```

Comment:
@dimitar-kostadinov , we need to follow-up if we can use this Extension resource instead of `./local-setup/operator-extension-resource.yaml` for the operator local setup we have - see [Alternative Setup Using the `gardener-operator` Local Setup](https://github.com/gardener/gardener-extension-registry-cache/blob/v0.17.1/docs/development/getting-started-locally.md#alternative-setup-using-the-gardener-operator-local-setup).